index.md

```diff
@@ -17,139 +43,288 @@ layout: default
     </div>
     <div class="bounce"><i class="fa glyphicon glyphicon-menu-down"></i><i class="fa glyphicon glyphicon-menu-down"></i></div>
   </div>
-  <div class="software-wrapper">
-    <div class="container">
-      <div class="title-software-inner">
-        <h2>Types of software we develop</h2>
-        <section class="image-wrapper">
-          <img class="img-responsive" src="./assets/images/icon-01.png" alt="Web applications">
-          <h3>Web applications</h3>
-          <p>
-            Applications that run<br> responsive design, Security
-          </p>
-        </section>
-        <section class="image-wrapper">
-          <img class="img-responsive" src="./assets/images/icon-02.png" alt="Mobile applications">
-          <h3>Mobile applications</h3>
-          <p>
-            We develop for<br> IOS, Android, Web
-          </p>
-        </section>
-        <section class="image-wrapper">
-          <img class="img-responsive" src="./assets/images/icon-01.png" alt="Desktop applications">
-          <h3>Desktop applications</h3>
-          <p>
-            Windows desktops apps<br> macOS apps<br> Universal Windows Platform (UWP) apps.
-          </p>
-        </section>
-      </div>
+</section>
+<section class="content-services">
+<div class="software-wrapper">
+<div class="container">
+<div class="title-software-inner">
+## Types of software we develop
+<section class="image-wrapper">
+![Web applications](./assets/images/icon-01.png){: .img-responsive }
+### Web applications
+Applications that run<br /> responsive design, Security
+</section>
+<section class="image-wrapper">
+![Mobile applications](./assets/images/icon-02.png){: .img-responsive }
+### Mobile applications
+We develop for<br /> IOS, Android, Web
+</section>
+<section class="image-wrapper">
+![Desktop applications](./assets/images/icon-01.png){: .img-responsive }
+### Desktop applications
+Windows desktops apps<br> macOS apps<br> Universal Windows Platform (UWP) apps.
+</section>
+</div>
+</div>
+</div>
+<div class="software-wrapper">
+  <div class="container">
+    <div class="title-software-inner">
```

Comment:
This will still render as code blocks, because it's indented more than 3 spaces.